### ISSUE TITLE
Politoed not yet attched to Poliwhirl

### ISSUE THREAD

**Issue by Try4se** [pokebotred/pokebotwiki#21]
Politoed evolves from Poliwhirl in the core games by trading a Poliwhirl holding king's rock, based on how the rest of the trade evolutions work, it should be a level up evolution. 

https://www.serebii.net/pokedex-sm/186.shtml

![politoed](https://user-images.githubusercontent.com/54107998/63212798-9150bd00-c0d7-11e9-8adb-8d357c67d631.png)

**Comment by bausshf** [comment]
Yeah, it has been left out for a while because I'm unsure how to introduce it properly whether it should be random when evolving using stone or if it should have a separate stone or whether it should level to that if you don't do it by stone to Poliwrath.
